torch/nn/modules/transformer.py

```diff
@@ -46,28 +48,30 @@ class Transformer(Module):
     def __init__(self, d_model: int = 512, nhead: int = 8, num_encoder_layers: int = 6,
                  num_decoder_layers: int = 6, dim_feedforward: int = 2048, dropout: float = 0.1,
                  activation: str = "relu", custom_encoder: Optional[Any] = None, custom_decoder: Optional[Any] = None,
-                 layer_norm_eps: float = 1e-5) -> None:
+                 layer_norm_eps: float = 1e-5, batch_first: bool = False) -> None:
         super(Transformer, self).__init__()
 
         if custom_encoder is not None:
             self.encoder = custom_encoder
         else:
             encoder_layer = TransformerEncoderLayer(d_model, nhead, dim_feedforward, dropout, activation, layer_norm_eps)
```

Comment:
Sounds good. I will make the change according to your description later tonight.

Which branch should I the rebase to? The current `master`? I am seeing like a few hundreds in GitKraken...

Reply:
> Which branch should I the rebase to? The current master? I am seeing like a few hundreds in GitKraken...

Yep, current master is the one! Thanks!

Reply:
Just rebased with `master`. Now `Transformer` takes `b_f` and passes to encoder/decoder layers instead of the wrapper. Accordingly the tests in `test_nn.py` have been changed. @jbschlosser